diesel/src/macros/mod.rs

```diff
@@ -989,6 +989,17 @@ macro_rules! __diesel_table_query_source_impl {
 ///
 /// assert_eq!(implicit_on_clause_sql, explicit_on_clause_sql);
 /// # }
+///
+/// In the example above, in line `joinable!(posts -> users (user_id));` specifies the relation of
+/// the tables and the ON clause in the following way:
+///
+/// `table_with_foreighn_key -> table_that_foreign_key_points_to (foreign_key)`
+/// and the Sql generated is:
```

Comment:
I would use a different formulation for this part of the documentation:

For joins that do not explicitly use on clauses via [`JoinOnDsl`](link to JoinOnDsl) the following on clause is generated implicitly:
```sql
post JOIN users ON posts.user_id = users.id
```

Reply:
That's way better worded and informative, I will go ahead and replace it, thanks!